app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -216,6 +218,7 @@ public void onConnectFailure(Call call, CallException error) {
             @Override
             public void onConnected(Call call) {
                 setAudioFocus(true);
+                soundPoolManager.getInstance(VoiceActivity.this).stopRinging();
```

Comment:
I think there needs to be a stopRinging in onDisconnected in the event of a user initiated disconnected.